tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -0,0 +1,23 @@
+@api
+Feature: search
+  As a user
+  I want to do full text search
+  So that I can find the files with the content I am looking for
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+  Scenario: search a file using tag
+    Given using spaces DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "file3.txt"
+    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+      | tag1 |
+    And user "Alice" has created the following tags for file "file2.txt" of the space "Personal":
+      | tag1 |
```

Comment:
Can we say **added** rather than **created**? Because creating the same tag twice sounds odd to me.   
@SwikritiT @saw-jan @SagarGi

Reply:
true, it does sound weird. 

idk how this step works internally 
1. it creates a tag
2. it adds tag to a file

maybe we can split the steps to
"user "Alice" has created a tag <tag>"
and then added
is there a step already?

But maybe this can be done in another PR if it's a good idea

Reply:
Yes, that can be done in other PRs. But I highly doubt that adding tag to a file is two-step process.
CC @grgprarup

Reply:
AFAIK we cannot create a tag only. According to API docs, we can assign the tags to resources and unassign the tags to resources, so using the term assign and unassign would be better. But every refactors can be done in separate PR.